src/Controller/Component/FlashComponent.php

```diff
@@ -42,7 +42,11 @@ class FlashComponent extends Component
     protected $_defaultConfig = [
         'key' => 'flash',
         'element' => 'default',
-        'params' => []
+        'params' => [],
+        'stacking' => [
+            'enabled' => false,
+            'limit' => 50
```

Comment:
I dis this to avoir session overflowing in case things go bas. But it can be removed